test/built-ins/Temporal/Calendar/prototype/monthDayFromFields/missing-properties.js

```diff
@@ -23,20 +23,21 @@ const missingDay = {
 };
 assert.throws(TypeError, () => instance.monthDayFromFields(missingDay), "day should be checked before year and month");
 
-let getMonthCode = false;
-let getYear = false;
-const monthWithoutYear = {
-  day: 1,
-  month: 5,
+let got = [];
+const fieldsSpy = {
+  get year() {
+    got.push("year");
+  },
+  get month() {
+    got.push("month");
+  },
   get monthCode() {
-    getMonthCode = true;
+    got.push("monthCode");
   },
-  get year() {
-    getYear = true;
+  get day() {
+    got.push("day");
+    return 1;
   },
 };
-assert.throws(TypeError, () => instance.monthDayFromFields(monthWithoutYear), "year/month should be checked after fetching but before resolving the month code");
```

Comment:
This assertion is no longer appropriate because `monthDayFromFields({ month, day })` is acceptable in the ISO 8601 calendar, but wasn't actually material to the test so has been replaced with assertions against a full complement of property access spies that also subsume `{ day: 1 }`.